src/wp-admin/includes/upgrade.php

```diff
@@ -2397,7 +2400,41 @@ function upgrade_650() {
 		wp_set_option_autoload_values( $autoload );
 	}
 }
+/**
+ * Executes changes made in WordPress 6.7.0.
+ *
+ * @ignore
+ * @since 6.7.0
+ *
+ * @global int  $wp_current_db_version The old (current) database version.
+ */
+function upgrade_670() {
+	global $wp_current_db_version;
+
+	if ( $wp_current_db_version < 57160 ) {
+		$options_ids = array(
+			'recently_activated',
+			'_wp_suggested_policy_text_has_changed',
+			'dashboard_widget_options',
+			'ftp_credentials',
+			'recently_edited',
+			'adminhash',
+			'nav_menu_options',
+			'wp_force_deactivated_plugins',
+			'delete_blog_hash',
+			'allowedthemes',
+			'admin_email',
+			'recently_activated',
+			'https_detection_errors',
+			'fresh_site',
```

Comment:
There is one more instance where `'fresh_site'` updates needs to be modified, in `wp-admin/includes/upgrade.php`.

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/6465/commits/28e9a8779a768d17544157b8d39bd71305e2d950